sourcegraph/cody
# Cody: Fix stop generating error

## Description

There is a case where a user aborts the chat before a message is sent. In that case we don't actually want an error. We can handle this for each individual case.

We now already do this for fixups, which this was originally added for:

https://sourcegraph.com/github.com/sourcegraph/cody@tr/fix-stop-gen-error/-/blob/vscode/src/chat/FixupViewProvider.ts?L101%3A1-104%3A10

<img width="726" alt="image" src="https://github.com/sourcegraph/cody/assets/9516420/f060f07f-3ac0-4e19-8891-1fb15a952c1d">

## Test plan

- Check no error when "Stop generating" is clicked
- Check no error when fixup is generated with incorrect xml tag
